common/common.go

```diff
@@ -26,6 +26,7 @@ const (
 	DefaultNumE2eProbes       = 50
 	DefaultMinTTL             = 1
 	DefaultMaxTTL             = 30
+	DefaultE2eProbeTTL        = 200
```

Comment:
I'm wondering if we should just re-use the same "maxTTL" values used by traceroute for e2e probe.
Since if 30 (or whatever configure by user) is enough to reach destination for traceroute, it should be also enough to reach destination for e2e probe right? it seems we don't need 2 different max TTL 🤔

So, maybe we can just re-use existing max-ttl (default and client/CLI provided values) and if needed later, we can have a different config/option. 

WDYT?